README.md

```diff
@@ -16,20 +16,38 @@ If bundler is not being used to manage dependencies, install the gem by executin
 
 ### Client
 
-Customer.io API client is initialized with an API key which is used for bearer token authentication.
+#### ClientV1
+
+ClientV1 is used to retreive data for the following resources:
+
+- Customer
+- CustomerioObject
+- ObjectRelationship
+
+ClientV1 is initialized with an API key which is used for bearer token authentication.
 
 ```ruby
-client = CustomerioAPI::Client.new(api_key: ENV['CUSTOMERIO_API_KEY'])
+client_v1 = CustomerioAPI::ClientV1.new(api_key: ENV['CUSTOMERIO_API_KEY'])
+```
+
+#### ClientV2
+
+ClientV2 is used to perform various operations based on the type (person, object or delivery) and action (identify, add_relationships, delete, etc) that you set in the request.
+
+ClientV2 is initialized with a site_id and track_api_key which is used for basic authentication.
+
+```ruby
+client_v2 = CustomerioAPI::ClientV1.new(site_id: ENV['SITE_ID'], track_api_key: ENV['TRACK_API_KEY'])
```

Comment:
I think naming it to V1Client sounds better lol
```suggestion
v1_client = CustomerioAPI::V1Client.new(site_id: ENV['SITE_ID'], track_api_key: ENV['TRACK_API_KEY'])
```

Reply:
lol sure 👍